waterArduino.INO

```diff
@@ -22,28 +30,37 @@ void setup()
 
 void loop() 
 {
-  for(int i = 0; i < 10; i++)   //for loop to calculate average of ten soil readings
+  avgDirt = analogRead(sensPin);
```

Comment:
we have to set `avgDirt` to something at least once before we can add to it inside the loop.